front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftSections/HomepageBanner/index.tsx

```diff
@@ -262,6 +322,35 @@ const HomepageBannerSettings = () => {
           );
         }}
       />
+
+      <Label htmlFor="bannerImage">{formatMessage(messages.bannerImage)}</Label>
+      {homepageSettings.banner_layout === 'fixed_ratio_layout' &&
+      imageFiles.length > 0 ? (
+        <ImageCropperContainer
+          image={imageFiles[0] || null}
+          onComplete={handleOnAdd}
+          aspectRatioWidth={3}
+          aspectRatioHeight={1}
+          onRemove={handleOnRemove}
+        />
+      ) : (
+        <ImagesDropzone
+          id="bannerImage"
+          images={imageFiles}
+          imagePreviewRatio={1 / 2}
+          maxImagePreviewWidth="360px"
+          objectFit="contain"
+          acceptedFileTypes={{
+            'image/*': ['.jpg', '.jpeg', '.png'],
+          }}
+          onAdd={(images) => {
+            setImageFiles(images);
+            handleOnAdd(images[0].base64);
+          }}
```

Comment:
General question about this ImagesDropzone component - can it accept multiple images? I feel like we only ever handle a single image with it? And the ImageFiles attribute is only ever an array with 1 file right? :thinking:

Reply:
Yes, this is correct. When the component was created, maybe the idea was that someday it would handle multiple images (I'm not sure) but it was designed in this way and now we are a little bit stuck with it. It would be a nice improvement to change it but it's used extensively so it would require some work.